src/test/java/Homework17.java

```diff
@@ -0,0 +1,20 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest{
+    @Test
+    public void addSongToPlaylist() throws InterruptedException{
+        String text ="Added 1 song into";
+
+      openUrl();
+        enterEmail("varjan80@mail.ru");
+        enterPassword("te$t$tudent");
+        clickSubmitBtn();
+
+        searchSong("Pluto");
+        viewAllBtn();
+        firstSong();
+        clickAddToBtn();
+        choosePlaylist();
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**